### ISSUE TITLE
How to train from scratch?

### ISSUE THREAD

**Issue by 21century-bio** [WxTu/DFCN#2]
Hi, thanks a lot for the code!

As mentioned in the paper, the training procedure includes 3 steps: "First, we pre-train the AE and IGAE independently for 30 iterations by minimizing the reconstruction loss functions. Then, both subnetworks are integrated into a united framework for another 100 iterations. Finally, with the learned centers of different clusters and under the guidance of the triplet self-supervised strategy, we train the whole network for at least 200 itera- tions until convergence."

I have trained AE and IGAE independently, but how to integrate them into a united framework for training without using the triplet strategy, i.e. the second step ? 

Thanks for your advice in advance!

**Comment by WxTu** [comment]
Thank you for your attention! We have uploaded the model pre-training demo. Kindly take a moment to review it.
